.editorconfig

```diff
@@ -170,7 +170,7 @@ csharp_new_line_between_query_expression_clauses = true
 
 # Identation options
 csharp_indent_case_contents = true
-csharp_indent_case_contents_when_block = true
+csharp_indent_case_contents_when_block = false
```

Comment:
If we're changing this, I assume it'll produce warnings on the build. This PR should resolve those warnings. My previous comment meant that if you change this and it produces those warnings, then a majority of these cases could just be fixed to no longer contain `{ }`. I'm not saying that this cannot be set to `false`. What I am saying is that setting this to `false` in this PR should contain the changes to remove the extra curly braces in this PR.